releases/release-7.4.0.md

```diff
@@ -0,0 +1,500 @@
+---
+title: TiDB 7.4.0 Release Notes
+summary: 了解 TiDB 7.4.0 版本的新功能、兼容性变更、改进提升，以及错误修复。
+---
+
+# TiDB 7.4.0 Release Notes
+
+发版日期：2023 年 x 月 x 日
+
+TiDB 版本：7.4.0
+
+试用链接：[快速体验](https://docs.pingcap.com/zh/tidb/v7.3/quick-start-with-tidb) | [下载离线包](https://cn.pingcap.com/product-community/)
+
+在 7.4.0 版本中，你可以获得以下关键特性：
+
+<table>
+<thead>
+  <tr>
+    <th>分类</th>
+    <th>功能</th>
+    <th>描述</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>可扩展性与性能</td>
+    <td>Enhance the performance for adding several indexes of a table in a single ALTER statement (experimental)</td>
+    <td>From v6.2 the user can add several indexes of a table in a single ALTER statement. However, the performance is the same as running two single add index DDL statements x, y, which used to take x-time +y-time, they now take significantly less.</td>
+  </tr>
+  <tr>
+    <td rowspan="3">稳定性与高可用</td>
+    <td>Improving the performance and stability of 'Import into' and 'Add Index' operations via `global sorting`</td>
+    <td>Before v7.4.0, tasks like ADD INDEX or IMPORT INTO in the distributed parallel execution framework required TiDB nodes to allocate local disk space for sorting data before importing it into TiKV. This approach, involving partial and localized sorting, often led to data overlaps, increasing TiKV's resource consumption and lower performance and stability. With the Global Sorting feature in v7.4.0, data is temporarily stored in S3 for global sorting. Then the data is imported into TiKV in an orderly, eliminating the need for TiKV to consume extra resources on compactions. This significantly enhances the performance and stability of operations like IMPORT INTO and ADD INDEX.</td>
+  </tr>
+  <tr>
+    <td>资源管控支持自动管理后台任务（实验特性）</td>
+    <td>从 v7.1.0 开始，资源管控成为正式功能，该特性有助于缓解不同工作负载间的资源与存储访问干扰。TiDB v7.4.0 将此资源控制应用于后台任务。资源管控可以识别和管理后台任务，例如自动收集统计信息、备份和恢复、LOAD DATA 以及在线 DDL。未来，所有后台任务都将纳入资源管控。</td>
+  </tr>
+  <tr>
+    <td>TiFlash supports storage-computing separation and S3</td>
+    <td>TiFlash introduces a cloud-native architecture as an option:
+      <ul>
+        <li>Disaggregates TiFlash's compute and storage, which is a milestone for elastic HTAP resource utilization.</li>
+        <li>Introduces S3-based storage engine, which can provide shared storage at a lower cost.</li>
+      </ul>
+    </td>
+  </tr>
+  <tr>
+    <td rowspan="2">SQL</td>
+    <td>More complete partition management</td>
+    <td>Prior to v7.4, TiDB supported truncate partition, exchange partition, add/drop/reorganize partition on Range/List Partitioning, and add/coalesce partition on Hash/Key partitioning
+In this version, TiDB partition management adds:
+      <ul>
+        <li>Remove partition</li>
+        <li>Partitioning existing non-partitioned tables</li>
+        <li>Modifying existing partition types on tables</li>
```

Comment:
<td><a href="https://docs.pingcap.com/tidb/v7.4/partitioned-table/convert-a-partitioned-table-to-a-non-partitioned-table" target="_blank">TiDB fully supports partition management</a> </td>
    <td>Before v7.4.0, Range/List partitioned tables support partition management operations such as <code>TRUNCATE</code>, <code>EXCHANGE</code>, <code>ADD</code>, <code>DROP</code>, and <code>REORGANIZE</code>, and Hash/Key partitioned tables support partition management operations such as <code>ADD</code> and <code>COALESCE</code>.
    <p>Now TiDB also supports the following partition management operations:</p>
	      <ul>
        <li>Convert partitioned tables to non-partitioned tables</li>
        <li>Partition existing non-partitioned tables</li>
        <li>Modify partition types for existing tables</li>